schemas/gdbots/common/mixin/labelable/1-0-0.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<pbj-schema xmlns="http://gdbots.io/pbj/xsd"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://gdbots.io/pbj/xsd http://gdbots.io/pbj/xsd/schema.xsd">
+
+  <schema id="pbj:gdbots:common:mixin:labelable:1-0-0" mixin="true">
+    <fields>
+      <field name="labels" type="string" rule="list">
+        <description>
+          Labels is a list that categorizes data or tracks references in
```

Comment:
@rsumilang should be a set and we probably should constrain the format.  maybe `[\w-]`.  rather than `ANy^T%h&ng*@!?`

Reply:
@rsumilang has search indexing implications when needing fulltext vs exact match (which any field with a pattern is automatically a term/exact match search)